README.md

```diff
@@ -41,28 +41,28 @@ sktime is a library for time series analysis in Python. It provides a unified in
 [roadmap]: https://www.sktime.net/en/latest/roadmap.html
 [related software]: https://www.sktime.net/en/latest/related_software.html
 
-## :speech_balloon: Where to ask questions
+## :speech_balloon: Where to ask questions!
 
-Questions and feedback are extremely welcome! Please understand that we won't be able to provide individual support via email. We also believe that help is much more valuable if it's shared publicly, so that more people can benefit from it.
+We welcome questions and feedback and appreciate your understanding that we are unable to offer personalized assistance via email. We strongly believe in the value of sharing help publicly, as it allows a wider audience to benefit from it.
```

Comment:
This makes the sentence more negative.

We aren't actually receiving that many emails, how about we just turn this into a neutral: "questions and feedback are extremely welcome!" then just pointing to the table without saying anything about email.

Reply:
how about this "We absolutely love hearing your questions and feedback! Just a friendly heads-up, though: we're unable to offer direct support through email. But fear not! We strongly believe that sharing help publicly is the way to go. By doing so, we can ensure that a larger community benefits from the knowledge and assistance provided."

Reply:
are you saying that we should remove the part after "questions and feedback are extremely welcome!"
sorry if I didn't get you.

Reply:
> are you saying that we should remove the part after "questions and feedback are extremely welcome!" 

Yes, no need to point out the part about email:

* because we actually answer emails (even if pointing to the issue tracker)
* and people don't seem to get the wrong idea that email is the primary route for getting help

So let's just remove it!